server/render/index.js

```diff
@@ -68,12 +67,18 @@ export function render( element, key = JSON.stringify( element ) ) {
 	//todo: render an error?
 }
 
-export function serverRender( context ) {
-	if ( config.isEnabled( 'server-side-rendering' ) ) {
+export function serverRender( req, res ) {
```

Comment:
Good spot -- I think it actually should, since it's now also used for non-iso-routes. Meaning I should move the `res.render()` stuff out of that `config.isEnabled()` conditional.